Assets/Mirror/Runtime/ClientScene.cs

```diff
@@ -684,9 +685,8 @@ internal static void OnUpdateVarsMessage(UpdateVarsMessage msg)
 
             if (NetworkIdentity.spawned.TryGetValue(msg.netId, out NetworkIdentity localObject) && localObject != null)
             {
-                NetworkReader networkReader = NetworkReaderPool.GetReader(msg.payload);
-                localObject.OnUpdateVars(networkReader, false);
-                NetworkReaderPool.Recycle(networkReader);
+                using (NetworkReader networkReader = NetworkReaderPool.GetReader(msg.payload))
```

Comment:
{} too please. you used them in the code above as well

Reply:
See answer above